media-libs/libcamera/libcamera-0.6.0.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3
+inherit meson
+
+DESCRIPTION="A complex camera support library for Linux, Android, and ChromeOS"
+HOMEPAGE="https://libcamera.org/"
+
+# Official mirror, check https://libcamera.org/contributing.html
+EGIT_REPO_URI="https://git.linuxtv.org/libcamera.git"
+EGIT_COMMIT="v${PV}"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+
+DEPEND="
+	dev-libs/libyaml:=
+	dev-python/jinja2
+	dev-python/ply
```

Comment:
I suspect these aren't right unless CLI tools aren't being used for all of them. Please see the [Python guide](https://projects.gentoo.org/python/guide/).

Reply:
jinja2, ply and pyyaml are used during buildtime to manipulate sources see : utils/codegen/ipc/meson.build and utils/codegen/meson.build. In this case you **must** use python-any-r1 and move these deps into BDEPEND.

Nota: the package also allows optional python bindings (see src/py) that have to be handled by a runtime eclass (python-r1 or python-single-r1). I don't know if it's used for now and maybe it could be added later.